Extensions/bezier/package.lisp

```diff
@@ -0,0 +1,26 @@
+
+(defpackage :mcclim-bezier
+  (:use #:clim #:clim-lisp #:climi)
+  (:export #:bezier-design
```

Comment:
I have mixed feelings regarding `USE`ing `CLIM` package (it's OK though).

We shouldn't `USE` `CLIM-INTERNALS` package, so are references to it are explicit. It will be easier to browse the code and find parts which go beyond CLIM spec / need to be fixed with CLIM-INTERNALS change some internal detail.